pkg/gogocodec/codec_test.go

```diff
@@ -32,8 +32,8 @@ func TestCodecMarshallAndUnmarshall_jaeger_type(t *testing.T) {
 	s1 := &model.Span{OperationName: "foo", TraceID: model.NewTraceID(1, 2)}
 	data, err := c.Marshal(s1)
 	require.NoError(t, err)
-
 	s2 := &model.Span{}
+
```

Comment:
why these changes?

Reply:
Because while I was doing these changes, I was two commits behind the main. I was manually resolving conflicts by comparing changes. But by mistake, I might have forgotten.

removing it now